posts/multiprocessing-vs-multithreading-python.md

```diff
@@ -0,0 +1,137 @@
+
+# Multiprocessing vs. Multithreading in Python: What you need to know.
+
+***TLDR:*** If you don't want to understand the under-the-hood explanation, here's what you've been waiting for: you can use `threading` if your program is network bound or you are running your UI in Python using something like TkInter and `multiprocessing` if it's CPU bound.
```

Comment:
This sentence reads weird to me. Maybe just say:

> if your program is network bound or `multiprocessing` if it's CPU bound.

Reply:
Agree it reads less nicely. Was something I found out that wasn't written about on other blogs.